index.js

```diff
@@ -21,7 +28,9 @@ function intoRels (acc, x) {
 function createObjects (acc, p) {
   // rel="next" => 1: rel 2: next
   var m = p.match(/\s*(.+)\s*=\s*"?([^"]+)"?/)
-  if (m) acc[m[1]] = m[2];
+  if (m) {
+    acc[m[1]] = m[2];
+  }
```

Comment:
Why should this be a one liner and line 14 isn't flagged similarly?  Should I change line 14 as well?